edx_toggles/toggles.py

```diff
@@ -0,0 +1,44 @@
+from abc import ABC
+
+
+from django.conf import settings
+
+
+class BaseToggle(ABC):
+    def __init__(self, name, default=False):
+        self.name = name
+        self.default = default
+
+    def is_enabled(self):
+        raise NotImplementedError
+
+
+class ListedTogglesMixin:
+    # Note that we will always on this dict by sorted keys, so it would make sense to use a sortedcontainer here, but
+    # it would introduce an additional dependency in this package
```

Comment:
Actually, OrderedDict will sort the keys in the order they were inserted in the dictionary, but will not sort their values. Since we usually want to iterate on keys in alphabetical order (for instance to display them in the docs), it would make sense to use [sortedcontainer](https://pypi.org/project/sortedcontainers/).

Reply:
I understand now.  Let's just kill this comment and leave the `sorted` call below. If this ever becomes an issue, we can improve the implementation.

Reply:
Actually, it may make more sense for the user to sort of needed, or add sorted into the name. But, I don’t thing this is a big deal either way. I don’t imagine this will be used for other purposes.